base/applications/clipbrd/clipbrd.c

```diff
@@ -4,15 +4,48 @@
  * PURPOSE:     Provides a view of the contents of the ReactOS clipboard.
  * COPYRIGHT:   Copyright 2015-2018 Ricardo Hanke
  *              Copyright 2015-2018 Hermes Belusca-Maito
+ *              Copyright 2023 Katayama Hirofumi MZ <katayama.hirofumi.mz@gmail.com>
  */
 
 #include "precomp.h"
+#include <shlobj.h> /* For CFSTR_... */
 
 static const WCHAR szClassName[] = L"ClipBookWClass";
 
 CLIPBOARD_GLOBALS Globals;
 SCROLLSTATE Scrollstate;
 
+static void InitGlobals(HINSTANCE hInstance)
+{
+    ZeroMemory(&Globals, sizeof(Globals));
+    Globals.hInstance = hInstance;
+    Globals.hFont = (HFONT)GetStockObject(OEM_FIXED_FONT);
+
+    /* Registered clipboard formats */
+    Globals.uCFSTR_FILECONTENTS          = RegisterClipboardFormat(CFSTR_FILECONTENTS);
```

Comment:
Does the Win2k3 clipboard support all these?

Reply:
![image](https://github.com/reactos/reactos/assets/2107452/71bcf224-0107-4980-8572-9fc0e80c4ea4)

This is Win2k3's clipbrd with copied some files. Some menu items are grayed.

Reply:
Deleted some formats.